src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/deciders/QueueHealthDecider.java

```diff
@@ -93,13 +99,16 @@ private void configureActionPriority() {
    */
   private Action computeBestAction(NodeKey esNode, ResourceEnum threadPool) {
     Action action = null;
-
-    for (String actionName : actionsByUserPriority) {
-      action =
-        getAction(actionName, esNode, threadPool, true);
-      if (action != null) {
-        break;
+    if (canUseMoreHeap(esNode)) {
```

Comment:
Indentation looks wrong here.

Reply:
```java
  private Action computeBestAction(NodeKey esNode, ResourceEnum threadPool) {
    Action action = null;
    if (canUseMoreHeap(esNode)) {
      for (String actionName : actionsByUserPriority) {
        action = getAction(actionName, esNode, threadPool, true);
        if (action != null) {
          break;
        }
      }
    } else {
      PerformanceAnalyzerApp.RCA_RUNTIME_METRICS_AGGREGATOR.updateStat(
          RcaRuntimeMetrics.NO_INCREASE_ACTION_SUGGESTED, NAME + ":" + esNode.getHostAddress(), 1);
    }
    return action;
  }
```

Reply:
I think the deletion and addition makes it look like it in the diff. This is code as it copied here. Can you check if the indentation looks right to you ?